package.json

```diff
@@ -2,8 +2,11 @@
   "name": "talent",
   "version": "0.0.1",
   "description": "Learners Guild Talent",
+  "main": "src/server/server.js",
   "scripts": {
-    "start": "webpack -d && webpack-dev-server",
+    "start": "NODE_ENV=production node src/server/server.js",
+    "start:dev": "nodemon src/server/server.js",
+    "start:react-dev": "webpack -d && webpack-dev-server",
```

Comment:
can you create a script to execute webpack in `watch` mode?

Reply:
@punitrathore - pushed fix - modifying existing start:react-dev script to watch since there's no need to *not* watch.